backend/token/payload.go

```diff
@@ -0,0 +1,20 @@
+package token
+
+import (
+	"time"
+)
+
+type Payload struct {
+	UserID   string    `json:"user_id"`
+	Email    string    `json:"email"`
```

Comment:
just curious why we need both userId and email for the payload, wouldn't having just one or the other be enough since they're both unique?

Reply:
idk just felt like we could encode it in the token as well, we did the same for hussle